modelci/cli/__init__.py

```diff
@@ -1,57 +1,27 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-"""
-Author: Li Yuanming
-Email: yli056@e.ntu.edu.sg
-Date: 10/12/2020
-"""
-import click
-
-from modelci.app import (start as app_start, stop as app_stop)
-from modelci.cli import model_cli
-from modelci.utils import Logger
-from modelci.utils.docker_container_manager import DockerContainerManager
-
-logger = Logger(__name__, welcome=False)
+#  Copyright (c) NTU_CAP 2020. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
 
+import click
+from modelci.cli.modelci_service import *
 
 @click.group()
-@click.version_option()
 def cli():
-    """A complete platform for managing, converting, profiling, and deploying models as cloud services (MLaaS)"""
```

Comment:
This line is for printing helper:
![image](https://user-images.githubusercontent.com/36268431/106717792-22548480-663b-11eb-8012-0b69c3d84745.png)